src/rules/banRule.ts

```diff
@@ -50,7 +50,7 @@ export class Rule extends Lint.Rules.AbstractRule {
               * the name of the function in an array with one element: \`["functionName"]\`
               * an object in the following format: \`{"name": "functionName", "message": "optional explanation message"}\`
             * banning methods:
-              * an array with the object name, method name and optional message: \`["functionName", "methodName", "optional message"]\`
+              * an array with the object name, method name and optional message: \`["objectName", "methodName", "optional message"]\`
```

Comment:
This is the only change that I explicitly made for this PR. The rest of the changes are auto-formatter changes.